presto-kafka/src/test/java/io/prestosql/plugin/kafka/util/JsonEncoder.java

```diff
@@ -1,44 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.prestosql.plugin.kafka.util;
-
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import kafka.serializer.Encoder;
-import kafka.utils.VerifiableProperties;
-
-import java.io.UncheckedIOException;
-
-public class JsonEncoder
```

Comment:
this its not used and its same as JsonSerializer

Reply:
please move this to a separate commit `Remove unused classes`

Reply:
done